ansibledriver/service/ansible.py

```diff
@@ -421,9 +421,11 @@ def v2_runner_on_failed(self, result, *args, **kwargs):
         Note: even when a loop is used (so v2_runner_item_on_failed/v2_runner_item_on_ok is called for each item) this function is called at the end, when all items have been attempted but one has failed
         """
         logger.debug("v2_runner_on_failed: task={0}, result={1}, task_fields={2}".format(result._task, result._result, result._task_fields))
-        # TODO: handle ignore_errors?
         self.failed_task = result._task.get_name()
-        if 'msg' in result._result and 'Timeout' in result._result['msg'] and 'waiting for privilege escalation prompt' in result._result['msg']:
+        if 'ignore_errors' in kwargs and kwargs.get('ignore_errors'):
+            logger.info('ignoring failed task ' + str(self.failed_task) + ' as ignore_error is true, failed: ' + str(result._result))
```

Comment:
please change in the message with ignore_errors instead of ignore_error

Reply:
updated